services/src/main/webapp/WEB-INF/service-context.xml

```diff
@@ -235,6 +236,21 @@
 	    <property name="operationsMessageSender" ref="operationsMessageSender" />
 	</bean>
 	
+	<bean id="solrUpdateJmsTemplate" class="org.springframework.jms.core.JmsTemplate">
+        <property name="connectionFactory" ref="jmsFactory" />
+        <property name="defaultDestinationName" value="activemq:queue:repository.solrupdate?asyncConsumer=true" />
```

Comment:
Replace the value with a property from the properties file.

Reply:
couldn't figure out where to do this, actually, and meant to ask you. i tried creating properties in a few different places, but spring couldn't find them

Reply:
I believe it is /etc/server.properties
Generally, the config files used for local testing are found in the /etc/ folder.
More specifically, to find this, at the top of this context file it is specifying that it is going to try to get properties from a parameter named `server.properties.uri`.  Next, if you find where the webapp is deployed to jetty (in admin/pom.xml) you can see where `server.properties.uri` is specified (or do a search for the property).  This file gets included over in `services/pom.xml` when the war file is built.